mora/service/orgunit.py

```diff
@@ -135,12 +138,12 @@ def edit(self, unitid, req):
             ))
 
         if keys.PARENT in data.keys():
```

Comment:
Although not relevant to this change, I'd say this is an anti-pattern. `if keys.PARENT in data:` is the appropriate syntax…

Reply:
You've mentioned that before. I'll run through them all and fix it.

Reply:
Thanks :)